### PR TITLE
Update Twilio.markdown

### PR DESCRIPTION
**Pull request by rtvanhook**
1. Clarify that additional action is needed after installing credentials (i.e., install the integration from settings).  2.  Clarify that restart is required (simply reloading the yaml doesn't work).  3.  Reference secrets for those (like me) who didn't yet know about them.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/twilio.markdown [hunk 1]
```diff
@@ -17,7 +17,7 @@ Calls are limited to 10 minutes and will play a short trial message before your
 
 ## Configuration
 
-To use this notification integration in your installation, add the following to your `configuration.yaml` file:
+To use this notification integration in your installation, you must first install the base integration from Settings _after_ adding your Twilio credentials to your `configuration.yaml` file (either directly or using [secrets](https://www.home-assistant.io/docs/configuration/secrets/)):
```
Comment (frenck) [inline source/_integrations/twilio.markdown:20]: I see the confusion. We need to insert the configuration flow snippet the provides the instructions on how to set it up from the UI (which is currently missing).

Also, the reference to secrets isn't relevant here and should be removed.

File: source/_integrations/twilio.markdown [hunk 2]
```diff
@@ -39,7 +39,7 @@ auth_token:
 
 ### Usage
 
-After configuring the base Twilio integration, add and configure either or both of the [Twilio SMS](/integrations/twilio_sms) and [Twilio Phone](/integrations/twilio_call) integrations to utilize the notification functionality.
+After adding your Twilio account credentials to `configuration.yaml` and the Twilio base integration from Settings, you should restart Home Assistant.   Then, you can configure either or both of the [Twilio SMS](/integrations/twilio_sms) and [Twilio Phone](/integrations/twilio_call) integrations to utilize the notification functionality.
```
Comment (frenck) [inline source/_integrations/twilio.markdown]: There is no restart needed after setting things up from the UI. There is, however, a restart needed when changing YAML configuration. There is an order issue in your description.

```suggestion
After adding your Twilio account credentials to `configuration.yaml`, restart Home Assistant. Next, add the Twilio base integration. Then, you can configure either or both of the [Twilio SMS](/integrations/twilio_sms) and [Twilio Phone](/integrations/twilio_call) integrations to utilize the notification functionality.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/twilio.markdown (hunk 1 above)
Comment (rtvanhook) [reply source/_integrations/twilio.markdown:20]: New to Gihub, and it's kicking my ass.  How about:  "1) add your Twilio credentials to `configuration.yaml`, and then 2) install the base integration".  I see your point about secrets.  But documentation is for new users and not expereienced ones.  I'm new to HA, didn't yet know about HA secrets, lost a bit of time trying to figure it out.  I was hoping to leave a breadcrumb for the next "secrets unfamiliar" user.

**Comment by frenck** [comment]
Closing this PR as it is running stale while changes have been requested.

Feel free to re-open when you are ready to work on it again.

../Frenck

**Pull request closed without merging**